### PR TITLE
Update README for new (0.2) screenshots;

### PR DESCRIPTION
**Pull request by richb-hanover**
Add new screenshots, showing "Crusader Network Tester 0.1";
Update README and CHANGELOG to pass markdownlint with minimal warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,19 +11,21 @@ The **Crusader Network Tester** measures network rates and latency
 in the presence of upload and download traffic.
 It also incorporates a continuous latency tester for
 monitoring background responsiveness.
-It produces plots of the traffic rates,
-latency and packet loss.
-Crusader only uses TCP and UDP ports 35481 for its tests. Local server discovery uses UDP port 35483. The remote web server uses TCP port 35482.
+It produces plots of the traffic rates, latency and packet loss.
+Crusader uses TCP and UDP ports 35481 (only) for its tests.
+The Remote web server option uses TCP port 35482.
```
Comment (Zoxc) [inline README.md:16]: Remote -> remote

**Comment by Zoxc** [comment]
I'd avoid formatting and changes in the same commit. You could probably drop the screenshots as they're already out of date.

File: README.md [hunk 2]
```diff
@@ -11,19 +11,21 @@ The **Crusader Network Tester** measures network rates and latency
...
+Local server discovery uses UDP port 35483.
 
 **Pre-built binaries** for Windows, Mac, Linux,
 and Android are available on the
-[Releases](https://github.com/Zoxc/crusader/releases) page. The GUI is not prebuilt for Linux and must be built from source.
+[Releases](https://github.com/Zoxc/crusader/releases) page.
+The GUI is not prebuilt for Linux and must be built from source.
+A **Docker container** for running the server may be found on
```
Comment (Zoxc) [inline README.md:23]: Missing space here

File: README.md [hunk 3]
```diff
@@ -11,19 +11,21 @@ The **Crusader Network Tester** measures network rates and latency
...
+[dockerhub](https://hub.docker.com/r/zoxc/crusader).
 
-A **Docker container** for running the server may be found on [dockerhub](https://hub.docker.com/r/zoxc/crusader).
-
-**Status:** The latest Crusader release is linked at the top.
-  See the [CHANGELOG.md](./CHANGELOG.md)
-  file for details.
+**Status:** The latest Crusader release version is shown above.
```
Comment (Zoxc) [inline README.md:26]: This line could probably just be removed.

File: README.md [hunk 4]
```diff
@@ -35,29 +37,34 @@ a **client** that initiates the test.
...
-  [Command-line options](#command-line-options) section.
+  [Command-line options](./#command-line-options) section.
 
 * **Server tab**
   Runs the Crusader server, listening for connections from other clients
 
+* **Remote tab**
+  Starts a webserver (default port 35482).
+  A browser that connects to that port can initiate a test
+  against a remote Crusader server.
```
Comment (Zoxc) [inline README.md:66]: `a remote Crusader server` to `a Crusader server.` to avoid confusion?

File: README.md [hunk 5]
```diff
@@ -165,69 +174,95 @@ crusader remote
...
   Binaries are saved in the _src/target/debug_ directory
 
-* To get the git commit hash of the current checkout,
-  use `git rev-parse --short master`
-
-* I sometimes see this message: 
-`Warning: Load termination timed out. There may be residual untracked traffic in the background.` 
+* Current binaries display the full commit hash in the log files.
+  To get the git commit hash of the current checkout,
+  use `git rev-parse --short master`.
```
Comment (Zoxc) [inline README.md:253]: `git rev-parse HEAD` would be more correct.

File: README.md [hunk 6]
```diff
@@ -165,69 +174,95 @@ crusader remote
...
+  the OS requires you to use **System Preferences -> Security**
+  to permit Crusader to run.
+  
 * Crusader requires that TCP and UDP ports 35481 are open for its tests.
   Check that your firewall is letting those ports through.
-
+  Crusader also uses ports 35482 for the remote webserver
+  and port 35483 for discovering other Crusader Servers.
+  
+* The [Releases](https://github.com/Zoxc/crusader/releases) page
```
Comment (Zoxc) [inline README.md:243]: This doesn't seem to belong in troubleshooting?

File: README.md (hunk 3 above)
Comment (richb-hanover) [reply README.md:26]: When I'm a newcomer to any project, the second or third thing I look at is the current release info. I think this should remain as a pointer of where to find it.

File: README.md [hunk 7]
```diff
@@ -165,69 +174,95 @@ crusader remote
 * **`--plot-height <PIXELS>`**
 * **`--plot-title <PLOT_TITLE>`**
 * **`--latency-peer <LATENCY_PEER>`**
-          Specifies another server (peer) which will
-          also measure the latency to the server independently of the client
+  Specifies the IP address of another Crusader server (peer).
+  During the test, Crusader measures the latency to this peer
```
[the remark this answers is not in the document]
Comment (richb-hanover) [reply README.md:178]: I wasn't clear what this does, either. Would you provide better wording? Thanks.

**Comment by richb-hanover** [comment]
re: Formatting and screen shots... The screen shots match the released (0.1.0) version. This allows newcomers to get the right idea about the program.

But Crusader is moving fast (that's GREAT! And you're addressing my comments. That also makes me feel terrific!) I am content to "play catchup" for the next couple development cycles. I wouldn't mind a general "roadmap" of additional changes you you plan to make. Thanks.

**Comment by richb-hanover** [comment]
@Zoxc I think this README has caught up to the 0.2 version. Let me know if it needs tweaks. Thanks

File: media/Crusader-Results.png
Comment (Zoxc) [inline media/Crusader-Results.png:1]: This is now unused.

**Comment by richb-hanover** [comment]
Good catch. I removed both Crusader-Results.png and Crusader-Latency.png - both were renamed.

File: README.md [hunk 8]
```diff
@@ -96,25 +103,27 @@ that indicate times when packets were lost.
...
+```sh
 crusader test <server-ip>
 ```
 
 ### Remote
 
-To host a web server allowing remote control of a Crusader client, run this:
+To host a web server allowing remote control of a Crusader client,
+run this command, then connect to
+[http://this-ip-address:35482](http://this-ip-address:38452)
```
Comment (Zoxc) [inline README.md:124]: Not sure if this should be a link..

Comment (richb-hanover) [reply README.md]: Yes, you're right. Updated.

File: media/Crusader Screen Shots.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
+# Crusader Screen Shots
+This is a test page for the screen shots
```
Comment (Zoxc) [inline media/Crusader Screen Shots.md:2]: I'm a bit confused about the purpose of this.

Comment (richb-hanover) [reply media/Crusader Screen Shots.md:2]: I use this page to check the screen shots. The process I use is to get an image (screen shot) then trim off the drop shadow that the Mac captures. I then save the image in the _media_ directory. This page lets me ensure that the image retains the window border, but not the additional drop shadow. (See the current README that has the distracting drop shadow that I now trim off.)

File: README.md [hunk 10]
```diff
@@ -171,65 +180,90 @@ crusader remote
...
-* To get the git commit hash of the current checkout,
-  use `git rev-parse --short master`
-
-* I sometimes see this message: 
-`Warning: Load termination timed out. There may be residual untracked traffic in the background.` 
+* Current binaries display the full commit hash in the log files.
+  To get the git commit hash of the current checkout,
+  use `git rev-parse HEAD`.
+  
+* "I sometimes see this message. Should I be concerned?"
```
Comment (Zoxc) [inline README.md:256]: What's with the quotes here?

File: media/Crusader Screen Shots.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
+# Crusader Screen Shots
+
+I use this page to check the appearance of the screen shots.
```
Comment (Zoxc) [inline media/Crusader Screen Shots.md:3]: Maybe phrase this as how it's used as it's not clear who `I` is?

**Comment by richb-hanover** [comment]
Thanks!

**Pull request merged**